src/MetabaseTab/MetabaseTab.tsx

```diff
@@ -62,13 +72,19 @@ export const MetabaseTab: FunctionComponent<MetabaseTabProps> = ({ refreshNonce,
     webview!.addEventListener('dom-ready', () => {
       (webview as any).insertCSS(CUSTOM_CSS);
     });
-  }, [startUrlPath]);
+  }, [metabaseReady, startUrlPath]);
```

Comment:
Side effect of this is that metabase will be greyed out if the service goes out, and will also reload once it's back online.

Reply:
oh that's nice, so actually you're implementing #151 in the "disable when it suddenly went offline" version discussed there.

You can de facto link this PR with that closed issue, for future reference.

Reply:
Since you linked the issue here it already shows there, so I guess nothing left to do.